src/storage/heap_file.c

```diff
@@ -721,7 +721,9 @@ static int heap_chkreloc_print_notfound (const void *ignore_reloc_oid, void *ent
 static DISK_ISVALID heap_chkreloc_next (THREAD_ENTRY * thread_p, HEAP_CHKALL_RELOCOIDS * chk, PAGE_PTR pgptr);
 
 static int heap_chnguess_initialize (void);
+#if defined(ENABLE_UNUSED_FUNCTION)
 static int heap_chnguess_realloc (void);
+#endif /* #if defined (ENABLE_UNUSED_FUNCTION) */
```

Comment:
Nit. `#if defined` 구문이 바로 위에 있기 때문에 주석을 통한 정보 제공이 불필요하다고 생각합니다.

```suggestion
#endif
```